models/articles.model.js

```diff
@@ -23,39 +23,64 @@ exports.checkExists = (colName, tableName, query, msg) => {
     }
 }
 
-exports.selectArticles = (articleId) => {
+exports.selectArticles = (articleId, topic, sort_by = "created_at", order = "desc") => {
```

Comment:
right now you are using this model for both selecting all articles and selecting articles by id.

split this into two models which handle those two cases separately.